subseas_flex_fcst/config-sample.yaml

```diff
@@ -16,6 +16,17 @@ forecast_var_filePattern: CFSv2_SubXPRCP_CCAFCST_var_Apr_*.txt
 obs_filePattern: CFSv2_SubXPRCP_CCAFCST_obs_Apr_*.txt
 hcst_filePattern: CFSv2_SubXPRCP_CCAFCST_xvPr_Apr_*.txt
 variable: Precipitation Anomaly
+
+#S,L, target dates options
+numberLeads: 4
+dataLead1: wk1
+lead1: 1
+dataLead2: wk2
+lead2: 8
+dataLead3: wk3
+lead3: 15
+dataLead4: wk4
+lead4: 22
```

Comment:
could this be a dictionary in the first place? or whatever nested structured are called in config files? And that might save you from needing a numberLeads that will be the size of that list.

Reply:
```
leads:
    wk1: 1
    wk2: 8
    wk3: 15
    wk4: 22
```